dockerfiles/PyTorchContainerDockerfile

```diff
@@ -0,0 +1,20 @@
+ARG CODE_VERSION
+FROM clipper/py-rpc:${CODE_VERSION}
+
+COPY clipper_admin/clipper_admin/python_container_conda_deps.txt /lib/
+
+RUN echo deb http://ftp.de.debian.org/debian jessie-backports main >> /etc/apt/sources.list \
+      && apt-get update --fix-missing \
+      && apt-get install -yqq -t jessie-backports openjdk-8-jdk \
+      && conda install -y --file /lib/python_container_conda_deps.txt \
+      && conda install pytorch torchvision -c soumith
```

Comment:
They created a PyTorch conda channel recently, so this should be `conda install pytorch torchvision -c pytorch`

Reply:
Done